tests/jenkins/requirements/tests.txt

```diff
@@ -0,0 +1,6 @@
+mozlog==3.4
```

Comment:
Let's move this from `tests/jenkins/requirements/tests.txt` to `tests/jenkins/requirements.txt`, since it will otherwise be the only file in the `requirements` directory.

Also, this will need a `# pyup: ignore file` on the first line of the file, or pyup-bot will very persistently try to update the dependencies in this file.